styleguide/source/assets/js/tables.js

```diff
@@ -10,12 +10,12 @@
 (function ($, Drupal) {
   Drupal.behaviors.responsiveTables = {
     attach: function(context, settings) {
-      $("th", context).each(function () {
-        var eq = $(this).index();
-        var child = eq + 1;
-        var label = $(this).text();
-        $("td:nth-child(" + child + ")").append("&nbsp;").attr("data-title", label).addClass("responsive");
+      $('table').basictable();
+
+      $('.ama__resource--schedules table').each( function() {
+        $(this).basictable('start');
```

Comment:
It would be helpful to comment that this enables the responsive behavior. It's unclear from `start` what that would do.

Reply:
@damontgomery the thin line is a chrome rendering issue. It doesn't appear in any other browser